lte/gateway/python/magma/mobilityd/tests/BUILD.bazel

```diff
@@ -0,0 +1,100 @@
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@python_deps//:requirements.bzl", "requirement")
+load("//bazel:python_test.bzl", "pytest_test")
+
+COMMON_TEST_DEPS = [
+    requirement("pytest"),
+    requirement("pytest-cov"),
+    requirement("coverage-lcov"),
+    "//lte/gateway/python/magma/mobilityd:mobilityd",
+]
+
+COMMON_TAGS = [
+    "lte",
+    "mobilityd",
+]
+
+pytest_test(
+    name = "ip_alloc_dhcp_test",
+    srcs = ["ip_alloc_dhcp_test.py"],
+    tags = COMMON_TAGS + [
+        "manual",
+        "sudo_test",
```

Comment:
Maybe we should have a file that defines all allowed values for flags for python unit tests? I can imagine this set of flags exploding as people get creative.

Reply:
@themarwhal Added bazel/testing.md with a little general test and coverage documentation and usage of tags. Is this what you meant?

Reply:
Yes the doc looks great! Thanks. I meant something like creating a .bzl file that creates all these common tags as enum/consts. That way developers can look at that file to see what tags are available.

Reply:
cc @nstng